### PR TITLE
Suggest using ls-servers before deploying

### PR DESCRIPTION
**Pull request by soniaconnolly**
It's a good idea to check if there are extra instances running or if there are issues with the current instances before kicking off a deploy.

### CODE REVIEW

File: _articles/appdev-deploy.md [hunk 1]
```diff
@@ -142,7 +142,11 @@ Staging used to be deployed by this process, but this was changed to deploy the
    ```bash
    cd identity-devops
    ```
-1. Recycle the IDP instances to get the new code, it automatically creates a new migration instance first.
+1. Check current server status, and confirm that there aren't extra servers running. If there are, scale in old instances before deploying.
+   ```bash
+   aws-vault exec prod-power -- ./bin/ls-servers -e prod
```
Comment (zachmargolis) [inline _articles/appdev-deploy.md:147]: another option is `asg-size` which tell us the target/expected sizes, and if a resize is already scheduled?
```suggestion
   aws-vault exec prod-power -- ./bin/ls-servers -e prod
   aws-vault exec prod-power -- ./bin/asg-size prod idp
```

**Review by zachmargolis — APPROVED** [review]
LGTM

**Comment by soniaconnolly** [comment]
> I seem to recall some existing or desired Slack alerting when the health pool size had been increased more time than what's reasonable.
> 
> Maybe a huddle discussion?

Adding this as an eng huddle topic sounds like a good idea.

File: _articles/appdev-deploy.md (hunk 1 above)
Comment (soniaconnolly) [reply _articles/appdev-deploy.md:147]: Saw this after merging! That's also a good idea. I can make a new PR.

**Comment by jmhooper** [comment]
This is a test comment for notification purposes.

**Pull request merged**